examples/vision/reptile.py

```diff
@@ -0,0 +1,248 @@
+"""
+Title: Few-Shot learning with Reptile
+Author: [ADMoreau](https://github.com/ADMoreau)
+Date created: 2020/05/21
+Last modified: 2020/05/29
+Description: Reptile meta learning algorithm implemented in Keras
+
+"""
+
+"""
+The Reptile algorithm (https://arxiv.org/abs/1803.02999) was developed by OpenAI to
+perform model agnostic meta-learning. Specifically, this algorithm was designed to
+quickly learn to perform new tasks with minimal training, a task known as few-shot
+learning. The algorithm works by taking metagradients from the training of mini-batches
+representing new tasks, and using these gradients for a meta training step.
+"""
+
+import numpy as np
+import random
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import layers
+import tensorflow_datasets as tfds
+
+"""
+## Define the Parameters
+"""
+
+learning_rate = 0.003
+meta_step_size = 0.25
+
+inner_batch_size = 25
+eval_batch_size = 25
+
+meta_iters = 2000
+eval_iters = 5
+inner_iters = 4
+
+eval_interval = 1
+train_shots = 20
+shots = 5
+classes = 5
+
+"""
+## Dataset Functions
```

Comment:
Please add basic data visualization: the reader should immediately understand what you're training on, concretely. This is what gives meaning to the code example.